tests/BenchmarkDotNet.IntegrationTests/InProcessTest.cs

```diff
@@ -220,6 +225,13 @@ public void InProcessBenchmarkAllCasesSupported()
         [UsedImplicitly(ImplicitUseTargetFlags.WithMembers)]
         public class BenchmarkAllCases
         {
+            [Params("a", "b")]
+#if NET8_0_OR_GREATER
+            public required string ParamProperty { get; set; }
```

Comment:
It might be better to put this in a separate type, then you can assert the value in the benchmark method.